test-framework/amazon-lambda/pom.xml

```diff
@@ -31,5 +31,9 @@
             <groupId>io.quarkus.http</groupId>
             <artifactId>quarkus-http-vertx-backend</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vertx</artifactId>
+        </dependency>
```

Comment:
Same question as for the lambda ext.

Reply:
Same answer, you need to give access to the Vert.x classes as they are not resolved by transitivity anymore.